### DISCUSSION TITLE
AI Deck 1.1 (TOP) + multi ranger deck (Bottom).

### DISCUSSION THREAD

**Discussion by cli48** [bitcraze/discussions#1264] [Ideas]
Hello,
I'm trying to make a model driven drone that communicates flight controls based on classifications made by a model ran on the GAP8 to the ARM MCU pilot. I understand that the multi ranger is traditionally on the top but I want to reverse and place at the bottom. (Is this possible?) If so, is there a template or example I can base my work off of?

**Comment by knmcguire** [comment]
Sure! that should be possible. I assume you have found the expansion deck documentation already? https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/

You still need to have the right orientation, so you can not reverse the deck physically, it still needs to be with the right symbols in the right way: https://www.bitcraze.io/documentation/tutorials/getting-started-with-expansion-decks/. This means that the top range sensor will be blocked.
